www/docs/client/tanstack-react-query/migrating.mdx

```diff
@@ -19,9 +19,21 @@ We're working on a codemod to help you migrate your existing codebase over to th
 Run our upgrade CLI:
 
 ```sh
+cd path/to/your/frontend
+
 npx @trpc/upgrade@next
 ```
 
+You can override some key options:
+
+```sh
+# Override the path to the file which exports your tRPC React Query hooks if it's different to `trpc.ts`
+npx @trpc/upgrade@next --trpcFile=src/path/to/api.ts
+
+# Override the name for the tRPC React Query hooks import if you're changing it from `trpc`
+npx @trpc/upgrade@next --trpcImportName=api
+```
```

Comment:
are these still needed? I thought @barelyhuman fixed this

Reply:
these should get picked up automatically, we don't really have tests for that though, so that might be something we add in one of the future PRs

Reply:
Interesting, I needed it when applying at work

Reply:
Line number 31 might still be necessary since my logic depends on paths used by ts.

and full path specification might still be needed , but this should be working on the local CLI, not sure if this was released outside the preview builds

Reply:
No problem taking out these docs anyway, just needs a bit of fleshing out when we find things to know